cli/src/cli.ts

```diff
@@ -26,15 +26,27 @@ export const cli = yargs
           alias: "o",
           string: true,
           describe: "output path",
+        })
+        .option("outputMdastJson", {
+          boolean: true,
+          describe: "whether to output mdast json files",
+        })
+        .option("outputRst", {
+          boolean: true,
+          describe: "whether to output rST files",
+        })
```

Comment:
It looks like these options aren't mutually exclusive. Should they be? Or are we OK with users potentially generating both?

Reply:
They are not currently mutually exclusive, no. If you look at makeProject.ts it will add any number of 'stringifiers' depending on which options you passed. This is handy if you want to generate rST but also inspect the Yokedast json for debugging.